reana_job_controller/app.py

```diff
@@ -247,16 +286,52 @@ def get_job(job_id):
             }
 
         :resheader Content-Type: application/json
-        :statuscode 200: no error - the list has been returned.
+        :statuscode 200: no error - the Job has been returned.
```

Comment:
"no error" sounds negative, rather formulate positive things positively 😄 for example "200 OK - the jo has been returned"